python/pycylon/pycylon/data/column.pyx

```diff
@@ -14,49 +14,35 @@
 
 # distutils: language = c++
 
-from typing import List
-from libcpp.string cimport string
-from libcpp cimport bool
-from pycylon.common.status cimport CStatus
-from pycylon.common.status import Status
 from libcpp.memory cimport shared_ptr, make_shared
-from libcpp.vector cimport vector
-from pycylon.ctx.context cimport CCylonContext
-from pycylon.ctx.context import CylonContext
 from pycylon.data.data_type cimport CDataType
-from pycylon.data.data_type cimport DataType
-from pycylon.data.data_type cimport CType
-from pycylon.data.data_type cimport CLayout
 from pyarrow.lib cimport CArray as ArrowCAarray
-from pyarrow.lib cimport CChunkedArray as ArrowCChunkedAarray
-from pyarrow.lib cimport pyarrow_unwrap_array
 from pycylon.data.column cimport CColumn
-from pycylon.api.lib cimport pycylon_unwrap_data_type, pycylon_wrap_data_type
-from pycylon.data.data_type import DataType
+from pycylon.api.lib cimport pycylon_wrap_data_type
 from pyarrow.lib cimport pyarrow_unwrap_array, pyarrow_wrap_array
-
+import pyarrow as pa
+import pycylon as pc
 
 cdef class Column:
+    def __cinit__(self, array: pa.Array = None):
```

Comment:
we could probably take the input as a list, a numpy array or an Arrow array. All 3 should be compatible and we can handle it inside the constructor.